packages/api/src/sequelize/migrations/2025-08-13_00_alter_tcm-encounter_add_has-cardiac-codes.ts

```diff
@@ -0,0 +1,41 @@
+import type { Migration } from "..";
+import { cardiacCodes } from "../../command/medical/tcm-encounter/cardiac-codes";
```

Comment:
we should not import any code on the migrations - [ref](https://www.notion.so/metriport/Development-Best-Practices-bb3574d3c83d46dc9914a9e5c5542ca7?source=copy_link#07627cc53d374c198bc180f2cae0cd8b)

migrations are fixed in time, the code changes

Reply:
right, good point. this seemed weird to me too